### PR TITLE
Fixed #562

### LINKED ISSUE #562: Extracells2 fluid interface support
the ec fluid interface has implemented the ITileStorageMonitorable class that can return IStorageMonitorable and that an IMEMonitor<IAEFluidStack> that is the normal fluidInventory in the ae storage

### PR DESCRIPTION
**Pull request by DrummerMC**
- Fluids from ae network will now be shown in logistics pipes via ME Fluid Interface

### CODE REVIEW

File: common/logisticspipes/proxy/specialtankhandler/AETankHandler.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		return true;
+	}
+
+	@Override
+	public boolean isType(TileEntity tile) {
+		return tile instanceof ITileStorageMonitorable;
+	}
+
+	@Override
+	public List<TileEntity> getBaseTilesFor(TileEntity tile) {
```
Comment (davboecki) [inline common/logisticspipes/proxy/specialtankhandler/AETankHandler.java:46]: This should contain the controller and not the interface. It is used to identify If LP is connected to the same fluid providing structure to support a provider pipe and a supplier pipe on the same structure without causing a loop.

File: common/logisticspipes/proxy/specialtankhandler/AETankHandler.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		}
+	}
+	
+	
+	//Disable fluids that don't show in the Fluid Terminal
+	private boolean isFluidVisible(Fluid fluid){
+		if(fluid == null)
+			return true;
+		try{
+			if(Loader.isModLoaded("extracells")){
```
Comment (davboecki) [inline common/logisticspipes/proxy/specialtankhandler/AETankHandler.java]: Please use a proxy instead of this "workaround".

**Comment by DrummerMC** [comment]
Ok I have made these changest

**Comment by davboecki** [comment]
The proxy is there so you don't need to use reflection. Access the extra cells stuff directly. And try to avoid those try catch without handling the exception. Other than that this looks good.

**Comment by DrummerMC** [comment]
ok I will make this in the next hour

**Comment by DrummerMC** [comment]
I have added the ec api and removed try catch

File: common/logisticspipes/proxy/specialtankhandler/AETankHandler.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+			IGridNode node = host.getGridNode(ForgeDirection.UNKNOWN);
+			if(node != null){
+				try{
+					DimensionalCoord coord = node.getGrid().getPivot().getGridBlock().getLocation();
+					TileEntity mainTile = coord.getWorld().getTileEntity(coord.x, coord.y, coord.z);
+					if(mainTile != null){
+						tiles.add(mainTile);
+						return tiles;
+					}
+				}catch(Throwable e){}
```
Comment (davboecki) [inline common/logisticspipes/proxy/specialtankhandler/AETankHandler.java]: What is this try catch needed for?

**Comment by DrummerMC** [comment]
Removed try catch and added null checks

**Pull request merged**